src/ethereum_clis/clis/eels_t8n.py

```diff
@@ -0,0 +1,139 @@
+"""
+EELS T8N transition tool.
+
+https://github.com/ethereum/execution-specs
+"""
+
+from abc import ABC, abstractmethod
+from typing import List
+
+from ethereum_clis import TransitionTool, TransitionToolOutput
+from ethereum_test_base_types import BlobSchedule
+from ethereum_test_forks import Fork
+from ethereum_test_types import Alloc, Environment, Transaction
+
+
+class EELST8NWrapper(TransitionTool, ABC):
```

Comment:
I think this might be unnecessary, we could perhaps leverage the `TransitionTool.default` to not even requiring a `evm_bin` in the first place.

Let me open a PR to this PR.